elf_reader.go

```diff
@@ -582,6 +598,13 @@ func (ec *elfCode) loadBTFMaps(maps map[string]*MapSpec) error {
 
 			maps[name] = mapSpec
 		}
+
+		// Drain the ELF section reader to make sure all bytes are accounted for
+		// with BTF metadata.
+		var b bytes.Buffer
+		if i, err := b.ReadFrom(rs); i > 0 || err != nil {
```

Comment:
Nit: why not `io.Copy(ioutil.Discard, rs)`?

Reply:
The subsequent error contains the unexpected bytes. On second thought, this might not be a good idea since it could be a large amount of data that would be printed to stdout in most cases.

Reply:
Done, no longer dumping the bytes but just reporting `i` instead. (btw, all of `ioutil` is now deprecated, using `io.Discard` instead)

Reply:
Looks like we still need ioutil for the previous stable Go. :shrug: